Assets/00_Content/Scripts/Player/PlayerComponent.cs

```diff
@@ -1,13 +1,134 @@
+using Extensions;
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Linq;
 using UnityEngine;
 using UnityEngine.InputSystem;
+using Vikings;
 
 namespace Player {
 	public class PlayerComponent : MonoBehaviour {
```

Comment:
Would it be cleaner to move all the brawling stuff into its own `PlayerBrawling` component?

Reply:
agreed